src/main/handlers/exportProjectHandler.ts

```diff
@@ -0,0 +1,45 @@
+import { BrowserWindow, dialog } from 'electron';
+import { Project } from '../../sharedTypes';
+import { exportEDL } from '../export';
+
+const getExportFilePath: (
+  mainWindow: BrowserWindow | null
+) => Promise<string> = async (mainWindow) => {
+  if (mainWindow === null) {
+    throw new Error('Main window not defined');
+  }
+
+  const dialogResponse = await dialog.showSaveDialog(mainWindow, {
+    filters: [{ name: '.edl Files', extensions: ['edl'] }],
+    buttonLabel: 'Export',
+    title: 'Export Project',
+    properties: ['createDirectory'],
+  });
+
+  if (dialogResponse.canceled) {
+    throw new Error('Dialog cancelled');
+  }
+
+  return dialogResponse.filePath as string;
+};
+
+const exportProjectToFile: (
+  filePath: string,
+  mainWindow: BrowserWindow | null,
+  project: Project
+) => Promise<void> = async (filePath, mainWindow, project) => {
+  project.exportFilePath = filePath;
+
+  exportEDL(mainWindow, project);
```

Comment:
Would be really cool if we could make it open up the folder containing the export in finder/explorer after the export is done:

https://stackoverflow.com/questions/43991267/electron-open-file-directory-in-specific-application

Reply:
Seems pretty cool but is out of scope of this feature for MVP. Will create a ticket in the backlog after checking with other PMs